source/includes/entityref/_corporateuser.md

```diff
@@ -2,6 +2,8 @@
 
 Read-only entity that represents an internal user at an organization.
 
+With appropriate access, you can add or update users. Adding users may incur additional user fees. For questions about your account and billing, please contact Support or your Account Manager.
```

Comment:
Same comments as before.

Reply:
Changed 'Support' to 'Bullhorn Support' to match client facing documentation.